include/meson.build

```diff
@@ -3,12 +3,21 @@
 tomlplusplus_dep = declare_dependency(include_directories: include_dir)
 
 if not is_subproject
-	import('pkgconfig').generate(
+	if host_machine.system() == 'freebsd'
+		import('pkgconfig').generate(
 		name: meson.project_name(),
 		description: 'Header-only TOML config file parser and serializer for C++',
-		install_dir: get_option('datadir')/'pkgconfig',
+		install_dir: join_paths('libdata' , 'pkgconfig'),
```

Comment:
You're repeating the same thing twice, which is pretty verbose, instead of defining `the_install_dir = join_paths(... , 'pkgconfig')` in your if, then passing that to pkgconfig.generate.

Anyway, if you remove the install_dir kwarg, Meson already handles freebsd for you and defaults to lib/pkgconfig on most systems, and libdata on FreeBSD.

Reply:
Good to know, I managed to miss that section in the documentation. Thanks!